easybuild/easyblocks/i/ifort.py

```diff
@@ -76,3 +76,14 @@ def sanity_check_step(self):
         custom_commands = ["which ifort"]
 
         IntelBase.sanity_check_step(self, custom_paths=custom_paths, custom_commands=custom_commands)
+
+    def make_module_req_guess(self):
+        """
+        Additional paths to consider for prepend-paths statements in module file
+        """
+        guesses = super(EB_icc, self).make_module_req_guess()
+        if LooseVersion(self.version) >= LooseVersion('2016'):
```

Comment:
@damianam Why exactly is this version check here? If the `include` dir was not there for older versions, don't bother, the existence of the directory is checked before the path is added to `$CPATH`

Reply:
It existed, but the headers needed for fortran 2008 were not there, so there is no point in adding that path, I guess.